crates/astria-cli/src/bridge/collect.rs

```diff
@@ -50,6 +50,9 @@ pub(super) struct Command {
     /// The eth address of the astria bridge contracts.
     #[arg(long)]
     contract_address: ethers::types::Address,
+    /// Whether the contract is an ERC20 contract.
+    #[arg(long)]
+    contract_is_erc20: bool,
```

Comment:
Could we get away with just `erc20` rather than `contract_is_erc20` to be succinct?  I don't think the user would be any more confused if we omit the prefix, and while `contract_is_erc20` seems more suitable as a Rust variable, I think the command-line flag would be fine as `--erc20`.

Not a big deal though.

(If this does get changed, we'll need to update the CHANGELOG entry to match.)

Reply:
Removed the setting (now the default behavior of the dependency with a fallback if the contract is missing that call).